src/WebJobs.Script.WebHost/Security/Authorization/Policies/AuthorizationOptionsExtensions.cs

```diff
@@ -48,19 +49,25 @@ public static void AddScriptPolicies(this AuthorizationOptions options)
                 {
                     if (c.Resource is AuthorizationFilterContext filterContext)
                     {
+                        string extensionName = filterContext.RouteData.Values["extensionName"]?.ToString();
```

Comment:
I'm also debating whether we should renamed to cover the fact this is only used by extensions and align with the fact this would support other levels aside from system

Reply:
Agreed. Renamed the policy. I kind of also want to rename the other policy to something like HostAdminInvoke = "Host.Admin.Invoke", but that'd touch a lot of files so maybe we should do it separately.